skins/laika/src/view_models/BankAccount.ts

```diff
@@ -0,0 +1,29 @@
+import { Validity } from './Validity';
+
+export interface BankAccount {
+    validity: {
+        [key: string]: Validity
+    },
+    values: {
+        [key: string]: string
+    }
+}
+
+export interface BankAccountData {
+    accountId: string
+    bankId: string
+}
+
+export interface BankAccountRequest {
+    validationUrl: string
+    requestParams: object
+}
+
+export interface BankAccountResponse {
```

Comment:
I'm ok with this type of interface but I think `[key: string]: string` is better for responses coming from the server because this is now tight to how the back end responds. In case we add something there this is going to start failing.

Reply:
I have updated the interface.